apps/etl/etl_tasks/pdc.py

```diff
@@ -5,21 +5,44 @@
 from apps.etl.extraction.sources.pdc.extract import (
     HAZARD_TYPE_MAP,
     Pagination,
-    PDCExtraction,
+    PDCExtractionMetadata,
+    PDCExtractionMetaDataType,
+    PDCExtractionV2,
     PdcHazardInputMetadata,
     Restriction,
 )
-from main.celery import CeleryQueue
+from apps.etl.models import ExtractionData
 from main.configs import etl_config
 
 
 @shared_task
 def extract_and_transform_pdc_data():
-    data_url = f"{etl_config.PDC_SENTRY_BASE_URL}/hp_srv/services/hazards/t/json/get_active_hazards"
-    PDCExtraction.task.s(data_url).apply_async()
+    data_url = f"{etl_config.PDC_SENTRY_BASE_URL}/hp_srv/services/hazards/t/json/search_hazard"
+    pdc_latest_extraction = ExtractionData.objects.filter(
+        source=ExtractionData.Source.PDC,
+    ).last()
+    if pdc_latest_extraction:
+        start_date = datetime.strptime(pdc_latest_extraction.created_at, "%Y-%m-%d %H:%M:%S.%f")
+    else:
+        start_date = datetime.strptime(str(etl_config.PDC_START_DATE), "%Y-%m-%d")
+
+    data = PdcHazardInputMetadata(
+        pagination=Pagination(page=1, pagesize=100),
+        restrictions=[
+            [
+                Restriction(searchType="GREATER_THAN", createDate=str(int(start_date.timestamp() * 1000))),  # type: ignore
+            ]
+        ],
+    ).model_dump()
+    PDCExtractionV2.init_extraction(
+        metadata=PDCExtractionMetadata(
+            extraction_args=data,
```

Comment:
Let's update the naming.
```suggestion
            hazard_input=Hazard_input_metadata,
```